Java/E-montazysta/src/main/java/com/emontazysta/controller/ClientController.java

```diff
@@ -28,22 +25,20 @@ public class ClientController {
     @GetMapping("/all")
     @Operation(description = "Allows to get all Clients.", security = @SecurityRequirement(name = "bearer-key"))
     public ResponseEntity <List<ClientDto>> getAll() {
-        return ResponseEntity.ok().body(clientService.getAll().stream()
-                .map(ClientMapper::clientToDto)
-                .collect(Collectors.toList()));
+        return ResponseEntity.ok().body(clientService.getAll());
     }
 
     @GetMapping("/{id}")
     @Operation(description = "Allows to get Client by given Id.", security = @SecurityRequirement(name = "bearer-key"))
     public ResponseEntity<ClientDto> getById(@PathVariable Long id) {
-        return ResponseEntity.ok().body(ClientMapper.clientToDto(clientService.getById(id)));
+        return ResponseEntity.ok().body(clientService.getById(id));
     }
 
-    @PostMapping
+    @PostMapping("/create")
```

Comment:
I think that we should use same mapping in every controller, so if we want to use POST at /create we should use it everywhere.